File: chatwoot/ruby/src/CreateAUserExample.rb
require "json"
require "chatwoot_client"

ChatwootClient.configure do |config|
    config.api_key["api_access_token"] = "PLATFORM_APP_API_KEY"
end

user_create_update_payload = ChatwootClient::UserCreateUpdatePayload.new

begin
    response = ChatwootClient::UsersApi.new.create_a_user(
        user_create_update_payload, # data
    )

    p response
rescue ChatwootClient::ApiError => e
    puts "Exception when calling UsersApi#create_a_user: #{e}"
end

File: chatwoot/python/src/CreateAUserExample.py
import json
from datetime import date, datetime
from pprint import pprint

from chatwoot_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"platformAppApiKey": "PLATFORM_APP_API_KEY"},
)

with ApiClient(configuration) as api_client:
    user_create_update_payload = models.UserCreateUpdatePayload(
    )

    try:
        response = api.UsersApi(api_client).create_a_user(
            data=user_create_update_payload,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling UsersApi#create_a_user: %s\n" % e)
